src/analysis/souffle/examples/simple_canSayDowngrades/ok_claim_propagates.dl

```diff
@@ -0,0 +1,82 @@
+#include "src/analysis/souffle/taint.dl"
+#include "src/analysis/souffle/may_will.dl"
+#include "src/analysis/souffle/authorization_logic.dl"
+#include "src/analysis/souffle/tag_transforms.dl"
+
+// An interface used for running and getting results from the policy verifier.
+// Returns as output the violatesPolicy relation, which indicates whether 
+// there were failures in a way that is easy to read across the Souffle C++ interface.
+
+.output violatesPolicy(delimiter=";")
+
+// Following arcs code converetd to datalog code below
+/* Run binary //src/frontends/arcs/xform_to_datalog:generate_datalog_program) 
```

Comment:
No action required. We plan to delete `//src/frontends/arcs/xform_to_datalog`. So, this may not be available anymore. When we remove that //src/frontends/arcs directory, we should add a link to the commit that removes it.

Reply:
Not sure what do you mean by no action here. Should I delete the comment now and add link to commit later?

Reply:
Will remove comment and point to CL after deletion of //src/frontends/arcs directory